influxql/ast.go

```diff
@@ -1589,6 +1609,15 @@ func (s *SelectStatement) FunctionCalls() []*Call {
 	return a
 }
 
+// FunctionCallsByPosition returns the Call objects from the query in the order they appear in the select statement
+func (s *SelectStatement) FunctionCallsByPosition() [][]*Call {
+	var a [][]*Call
```

Comment:
Correct.  I need to know the column position the call was at to properly fill in during the post processing stage.